panels/sharing/cc-sharing-panel.c

```diff
@@ -170,6 +174,13 @@ cc_sharing_panel_dispose (GObject *object)
       priv->screen_sharing_dialog = NULL;
     }
 
+  if (priv->content_sharing_dialog)
+    {
+      gtk_widget_destroy (priv->content_sharing_dialog);
+      priv->content_sharing_dialog = NULL;
+    }
+
+
   g_clear_object (&priv->sharing_proxy);
```

Comment:
Should also clear the companion app helper proxy

Reply:
Done below